pkg/worker/worker.go

```diff
@@ -28,10 +41,70 @@ func New(c Config) *Worker {
 	if c.Log == nil {
 		tracer.Panic(tracer.Mask(fmt.Errorf("%T.Log must not be empty", c)))
 	}
+	if c.Met == nil {
+		tracer.Panic(tracer.Mask(fmt.Errorf("%T.Met must not be empty", c)))
+	}
+
+	cou := map[string]recorder.Interface{}
+
+	{
+		cou[MetricTotal] = recorder.NewCounter(recorder.CounterConfig{
+			Des: "the total amount of worker handler executions",
+			Lab: map[string][]string{
+				"service": {"specta"},
+				"success": {"true", "false"},
+			},
+			Met: c.Met,
+			Nam: MetricTotal,
+		})
+	}
+
+	gau := map[string]recorder.Interface{}
+
+	his := map[string]recorder.Interface{}
+
+	{
+		his[MetricDuration] = recorder.NewHistogram(recorder.HistogramConfig{
+			Des: "the time it takes for worker handler executions to complete",
+			Lab: map[string][]string{
+				"handler": {"container", "endpoint", "keypair", "stack"},
+				"service": {"specta"},
+				"success": {"true", "false"},
+			},
+			Buc: []float64{
+				0.10, //  100 ms
+				0.15, //  150 ms
+				0.20, //  200 ms
+				0.25, //  250 ms
+				0.50, //  500 ms
+
+				1.00, // 1000 ms
+				1.50, // 1500 ms
+				2.00, // 2000 ms
+				2.50, // 2500 ms
+				5.00, // 5000 ms
+			},
```

Comment:
This is everyone's best guess, but this looks to me like it makes sense for the worker executions. Can adapt if need be.